megatron/arguments.py

```diff
@@ -648,6 +648,11 @@ def _add_data_args(parser):
                        '1) a single data path, 2) multiple datasets in the'
                        'form: dataset1-weight dataset1-path dataset2-weight '
                        'dataset2-path ...')
+    group.add_argument('--valid-data', nargs='*', default=None,
```

Comment:
```suggestion
    group.add_argument('--valid-data-path', nargs='*', default=None,
```

Please, let's use consistent naming. And if accepted there will be a need for a corresponding change later in `args.valid_data`.

Reply:
Ah yes for sure, missed this